examples/keyword.rs

```diff
@@ -0,0 +1,21 @@
+/// Demostrates how to fetch and set keywords
+/// 
+/// Usage: cargo run --example keyword <keyword> <value>
+/// Example: cargo run --example keyword decoration:rounding (prints value) 
+/// Example: cargo run --example keyword decoration:rounding  15 (sets value)
+
+use hyprland::keyword::Keyword;
+
+fn main() -> hyprland::Result<()> {
+    let args: Vec<_> = std::env::args().skip(1).collect();
+    let keyword = if args.len() == 0 { "general:border_size" } else { &args[0].as_str() };
+    let value = if args.len() > 1 { Some(&args[1]) } else { None };
+
+    match value {
+        // Here we change a keyword, in this case we are doubling the border size we got above
+        Some(val) => Keyword::set(keyword, &**val)?,
```

Comment:
whats the `&**` for?

Reply:
The `Keyword::set` function requires a referenced string but because I used Option I had to reference the string. The double asterisk de-references it again. Another way to do it would be to clone the string when passing it into the Option. I can change it if you want.